### PR TITLE
Updated readme and fixed some small bugs.

### PR DESCRIPTION
**Pull request by StarArawn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +68,7 @@ rsx! {
 You can easily declare your own custom widgets:
 ```rust
 #[widget]
-pub fn MyCustomWidget(children: Children) {
```
Comment (StarArawn) [inline README.md:76]: @MrGVSV Can you make sure I'm not missing anything here? Feel free to commit changes directly to this branch.

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Kayak UI is in the very early stages of development. Important features are miss
 - Basic widget and global state management
 - Input events (Mouse, Keyboard, Char)
 - Fast and accurate layouts using morphorm: https://github.com/geom3trik/morphorm
-- A few default widgets check out [kayak_widgets](./kayak_widgets)!
+- A few default widgets check out [src/widgets](./src/widgets)!
```
Comment (MrGVSV) [inline README.md:23]: Maybe?
```suggestion
- A few default widgets check out Kayak's [built-in widgets](./src/widgets)!
```

File: README.md [hunk 3]
```diff
@@ -73,7 +68,7 @@ rsx! {
 You can easily declare your own custom widgets:
 ```rust
 #[widget]
-pub fn MyCustomWidget(children: Children) {
+pub fn MyCustomWidget(props: MyCustomWidgetProps) {
     rsx! {
```
Comment (MrGVSV) [inline README.md:72]: I think the other way works but it uses the "hidden children," so maybe we get the children directly like this:
```suggestion
pub fn MyCustomWidget(props: MyCustomWidgetProps) {
    let children = props.get_children();
    rsx! {
```

File: README.md (hunk 1 above)
Comment (MrGVSV) [reply README.md:76]: Yep, left a few suggestions. You can merge or discard them if you like.

**Review by MrGVSV — APPROVED** [review]
Looks good! Only minor suggestions.

**Pull request merged**